File: 000/22./nsnt.py
def is_prime(z):
    if z < 2: return False
    if z == 2: return True
    if z % 2 == 0: return False
    for i in range (3, int(z**0.5) + 1, 2):
        if z % i == 0: return False
    return True

n = int(input("Nhap vao so tu nhien n: "))

if n < 2:
    n = 2

while not is_prime(n):
    n += 1
print(n)

File: 000/22./nsnt.cpp
#include <iostream>
#include <cmath>
using namespace std;

bool is_prime(int z) {
	if (z < 2) return false;
	if (z == 2) return true;
	if (z % 2 == 0) return false;
	for (int i = 3; i <= sqrt(z); i += 2) {
		if (z % i == 0) {return false;}
	}

	return true;
}

int main() {
	int n;

	cout << "Nhap vao so tu nhien n: ";
	cin >> n;

	while(!is_prime(n)) {
		n ++;
	}

	cout << n << endl;
	
	return 0;
}
